addons/web/static/src/js/views/search_panel.js

```diff
@@ -30,10 +32,12 @@ odoo.define("web/static/src/js/views/search_panel.js", function (require) {
                 expanded: {},
             });
             this.model = useModel("searchModel");
-            // this.env.exportable({
-            //     export: this.exportState.bind(this),
-            //     import: this.importState.bind(this),
-            // });
+            this.scrollTop = 0;
+
+            useSharedState(this, "SearchPanel", {
+                export: () => this.exportState(),
```

Comment:
I see the form of handlers is always the same. Could it be avoided to specify them?

Reply:
Done